udpa/core/v1/resource_locator.proto

```diff
@@ -66,10 +66,26 @@ message ResourceLocator {
   }
 
   // Directives provide information to data-plane load balancers on how UDPA
-  // resource names are to be resolved. For example, they may provide
-  // alternative resource locators for when primary resolution fails. Directives
-  // are not part of resource names and do not appear in a xDS transport discovery
-  // request.
+  // resource names are to be interpreted and potentially further resolved. For
+  // example, they may provide alternative resource locators for when primary
+  // resolution fails. Directives are not part of resource names and do not
+  // appear in a xDS transport discovery request.
+  //
+  // When encoding to URIs, directives take form:
+  //
+  // <directive name>=<string representation of directive value>.
```

Comment:
Suggest removing the trailing `.` here, just to make it clear that that's not expected to be present in the format.